e2e/supabase/features/index.spec.ts

```diff
@@ -0,0 +1,7 @@
+import { test, expect } from '@playwright/test';
+
+test('has title', async ({ page }) => {
+  await page.goto('/');
+
+  await expect(page).toHaveTitle(/Logflare | Cloudflare, Vercel & Elixir Logging/);
```

Comment:
we should navigate to supabase dashboard logs page instead, no? 
or are the additional specs coming in a followup?